crates/openvino/src/port.rs

```diff
@@ -0,0 +1,32 @@
+use crate::{try_unsafe, util::Result};
+use openvino_sys::{ov_output_const_port_t, ov_port_get_any_name};
+use std::ffi::CStr;
+
+///Struct for Port of a model.
+pub struct Port {
+    instance: *mut ov_output_const_port_t,
+}
+
+impl Port {
+    ///Get [`ov_output_const_port_t`] instance.
+    pub fn instance(&self) -> Result<*mut ov_output_const_port_t> {
+        Ok(self.instance)
+    }
+
+    ///Create a new [`Port`] from [`ov_output_const_port_t`].
+    pub(crate) fn new(instance: *mut ov_output_const_port_t) -> Result<Self> {
```

Comment:
No need to return a `Result` when there is no chance for an error.
```suggestion
    pub(crate) fn new(instance: *mut ov_output_const_port_t) -> Self {
```